sources/editor/Stride.GameStudio/StrideEditorPlugin.cs

```diff
@@ -82,11 +82,7 @@ public override void InitializeSession(SessionViewModel session)
             var thumbnailService = new GameStudioThumbnailService(session, settingsProvider, builderService);
             session.ServiceProvider.RegisterService(thumbnailService);
 
-            var strideDebugService = new StrideDebugService(session.ServiceProvider);
-            session.ServiceProvider.RegisterService(strideDebugService);
-
             GameStudioViewModel.GameStudio.Preview = new PreviewViewModel(session);
-            GameStudioViewModel.GameStudio.Debugging = new DebuggingViewModel(GameStudioViewModel.GameStudio, strideDebugService);
```

Comment:
Do we need to keep the `DebuggingViewModel` class if we remove it from being created?